man/man8/zfs.8

```diff
@@ -2896,38 +2944,54 @@ Print verbose information about the stream and the time required to perform the
 .sp
 .ne 2
 .na
-\fB\fB-n\fR\fR
+\fB\fB-d\fR\fR
 .ad
 .sp .6
 .RS 4n
-Do not actually receive the stream. This can be useful in conjunction with the \fB-v\fR option to verify the name the receive operation would use.
+Discard the first element of the sent snapshot's file system name, using the remaining elements to determine the name of the target file system for the new snapshot as described in the paragraph above.
 .RE
 
 .sp
 .ne 2
 .na
-\fB\fB-o\fR \fBorigin\fR=\fIsnapshot\fR
+\fB\fB-e\fR\fR
 .ad
 .sp .6
 .RS 4n
-Forces the stream to be received as a clone of the given snapshot. This is only valid if the stream is an incremental stream whose source is the same as the provided origin.
+Discard all but the last element of the sent snapshot's file system name, using that element to determine the name of the target file system for the new snapshot as described in the paragraph above.
 .RE
 
 .sp
 .ne 2
 .na
-\fB\fB-F\fR\fR
+\fB\fB-o\fR \fBorigin\fR=\fIsnapshot\fR
 .ad
 .sp .6
 .RS 4n
-Force a rollback of the file system to the most recent snapshot before performing the receive operation. If receiving an incremental replication stream (for example, one generated by \fBzfs send -R -[iI]\fR), destroy snapshots and file systems that do not exist on the sending side.
+Forces the stream to be received as a clone of the given snapshot.
```

Comment:
Seems a bit odd that this is the only line in the man page that wraps to 80 characters.  Is that deliberate?